src/v3/src/types/stepTransformer.ts

```diff
@@ -10,12 +10,16 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import { IdxTransactionWithNextStep } from './jsonforms';
+import { IdxTransaction } from '@okta/okta-auth-js';
+
 import { FormBag } from './schema';
 import { WidgetProps } from './widget';
 
-export type IdxStepTransformer = (
-  transaction: IdxTransactionWithNextStep,
-  formBag: FormBag,
-  widgetProps: WidgetProps,
-) => FormBag;
+type Options = {
+  transaction: IdxTransaction;
+  prevTransaction?: IdxTransaction;
+  formBag: FormBag;
+  widgetProps: WidgetProps;
+};
```

Comment:
Duplicate type, see comment regarding extracting to types folder.

Reply:
duplicate with which one?

Reply:
Oh, nvm, I thought it was a dup of https://github.com/okta/okta-signin-widget/pull/2632/files#diff-5ac20d92def8ca7ac2dd91703edab859e4af5aba8fd8c414ecee3060a4431e9bR23-R28 (same name, similar props).